### PR TITLE
chore:added dummy data

### PR DESCRIPTION
**Pull request by locbgiang**

### CODE REVIEW

File: src/state/reducers/transactionReducer.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  data: [
+    {
+      date: '2020-1-1',
+      vendor: 'Costco',
+      vendor_code: '87',
+      category: 'Household',
+      amount: 11.19,
+    },
+    {
+      date: '2020-1-2',
```
Comment (vtopham) [inline src/state/reducers/transactionReducer.js:11]: this date format is simply beautiful

File: src/state/reducers/userReducer.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 const initialState = {
-  userName: 'testName',
+  userName: 'fightclubguy123',
```
Comment (vtopham) [inline src/state/reducers/userReducer.js:2]: hahahaha

**Review by vtopham — APPROVED** [review]
This looks great. It will be instrumental in letting us get to work with plotly. Thanks for doing this, Loc!

**Review by imriven — APPROVED** [review]
This is looking great to me! Clear and concise!

**Review by chelsabeth — APPROVED** [review]
Code looks clean and gets the job done!

**Pull request merged**